src/coreclr/src/vm/ClrEtwAll.man

```diff
@@ -3364,11 +3421,16 @@
                            task="CLRMethod"
                            symbol="MethodLoad_V2" message="$(string.RuntimePublisher.MethodLoad_V2EventMessage)"/>
 
-                    <event value="159" version="0" level="win:Verbose" template="R2RGetEntryPoint"
+                    <event value="159" version="0" level="win:Informational" template="R2RGetEntryPoint"
                            keywords ="CompilationDiagnosticKeyword" opcode="MethodLoad"
                            task="CLRMethod"
                            symbol="R2RGetEntryPoint" message="$(string.RuntimePublisher.R2RGetEntryPointEventMessage)"/>
 
+                    <event value="160" version="0" level="win:Informational" template="R2RGetEntryPointStarted"
```

Comment:
```suggestion
                    <event value="160" version="0" level="win:Informational" template="R2RGetEntryPointStart"
```

Convention for ETW event naming is to use "Start" and "Stop" (in EventSource the naming triggers a specific behavior that pairs start/stop events, in our native events it has no behavioral effects but its nice to match the convention)